client/asset/interface.go

```diff
@@ -858,8 +858,15 @@ type Balance struct {
 	// Locked is the total amount locked in the wallet which includes but
 	// is not limited to funds locked for swap but not actually swapped yet.
 	Locked uint64 `json:"locked"`
-	// Other is a place to list custom balance categories.
-	Other map[string]uint64 `json:"other"`
+	// Other is a place to list custom balance categories. It is recommended for
+	// custom balance added here to have a translation and tooltip info in
+	// client/webserver/site/src/js/wallet.js#customWalletBalanceCategory
+	Other map[string]*CustomBalance `json:"other"`
+}
+
+type CustomBalance struct {
+	Amount uint64 `json:"amt"`
+	Locked bool   `json:"locked"`
```

Comment:
Semantically, `Locked` is a flag to indicate that this `Amount` is not included in (subtracted from) the wallet's `Available` and included in `Locked`.  Please doc on this struct and it's fields so devs know  how to use this type and the parent `Balance` type.